docs/docs/working-with-video.md

```diff
@@ -0,0 +1,156 @@
+---
+title: Working With Video*
+issue: https://github.com/gatsbyjs/gatsby/issues/12903
+---
+
+This is a stub. Help our community expand it.
+
+Please use the [Gatsby Style Guide](/contributing/gatsby-style-guide/) to ensure your
+pull request gets accepted.
+
+The discussion around best practices for video in Gatsby is also ongoing on Github, chime in with your ideas here: https://github.com/gatsbyjs/gatsby/issues/3346
+
+---
+
```

Comment:
I miss a heading here like TOC or `Overview`. But I don't find the right wording now

Reply:
I'm not sure what you mean. The heading for the page is right above it (Working with Video). We've talked about creating more consistent patterns for TOC, such as in #6896. But I don't think that should hold up this PR!

Reply:
I did mean a header for the TOC. 

But I did not find any doc with a header for TOC. Only some introducing sentence on some pages following by TOC or links to aubpages.